### PR TITLE
Feature/166415644 css in js and fouc fix

### PR DESCRIPTION
**Pull request by lingyun1010**
The `slick-dots` position style could not be `relative` to prevent up/down when we change the tabs.

The height settings should be defined in `slick-slider` rather than `slick-list` to prevent multiple cards overlap flash when we change the tabs.

I pass `containerHeight` and `sliderHeight` props individually and set`sliderSettings` as required for `adaptiveHeight`.

### CODE REVIEW

File: src/carousel/CarouselCardsRow.js [hunk 1]
```diff
@@ -55,7 +55,9 @@ CarouselCardsRow.propTypes = {
   imageIconHeight: PropTypes.string,
   hoverColour: PropTypes.string,
   sliderSettings: PropTypes.object.isRequired,
+  //To prevent species summary panel's bottom border move up/down when tabs change.
```
Comment (alfonsomunozpomer) [inline src/carousel/CarouselCardsRow.js:58]: :rocket:

File: src/carousel/CarouselCardsRow.js [hunk 2]
```diff
@@ -55,7 +55,9 @@ CarouselCardsRow.propTypes = {
...
   containerHeight: PropTypes.string,
+  //To prevent slick-slider's bottom dots move up/down when tabs change as adaptiveHeight is applied.
```
Comment (alfonsomunozpomer) [inline src/carousel/CarouselCardsRow.js:60]: :rocket:

**Review by alfonsomunozpomer — APPROVED** [review]
Good work!

File: src/carousel/CarouselCardsRow.js (hunk 2 above)
Comment (lingyun1010) [reply src/carousel/CarouselCardsRow.js:60]: 🎆

**Pull request merged**